src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  padding-top: 2px;
+}
+
+.search__bar {
+  font-family: Avenir, Helvetica, sans-serif;
+  margin: 10px 0;
+  width: 100%;
+  box-sizing: border-box;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big {
+  font-size: 16px;
+  height: 70px;
+  line-height: 70px;
+  padding-left: 62px;
+}
+
+.small {
+  padding-top: 2px;
+  font-size: 14px;
+  height: 42px;
+  line-height: 40px;
+  padding-left: 33px;
+}
+
+.search__bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
You've already set it (and `border-radius`) in `.search__bar`. No need to do it again

Reply:
the same for `focus`

Reply:
+